datamod/chapter_1/items/items/hearts_donut.lua

```diff
@@ -0,0 +1,9 @@
+return "hearts_donut", function(item)
+    item.heal_variants_battle = {
+        ["kris"] = 10, 
+        ["susie"] = 90, 
+        ["ralsei"] = 60, 
+        ["noelle"] = 40,
+        ["default"] = 60
```

Comment:
bc of other suggested changes, remove "default" here

```suggestion
        ["noelle"] = 40
```